MentoringHomeWork/HW4/Task4/sum4/sum4.cs

```diff
@@ -0,0 +1,93 @@
+namespace Task4.sum4;
+
+public class Sum4
+{
+    public void Task4()
+    {
+        var number = 0;
+        var result = 0;
+        var number2 = 0;
+
+        Getnumber();
+        GetOperator();
+        CheckSum();
+        return;
+
```

Comment:
Be careful with new lines - minor: delete useless spaces, there should be just one

Reply:
resolved